students/jemielity_kamil/lesson_05_lists/character_picture_grid.py

```diff
@@ -0,0 +1,14 @@
+grid = [['.', '.', '.', '.', '.', '.'],
+        ['.', 'O', 'O', '.', '.', '.'],
+        ['O', 'O', 'O', 'O', '.', '.'],
+        ['O', 'O', 'O', 'O', 'O', '.'],
+        ['.', 'O', 'O', 'O', 'O', 'O'],
+        ['O', 'O', 'O', 'O', 'O', '.'],
+        ['O', 'O', 'O', 'O', '.', '.'],
+        ['.', 'O', 'O', '.', '.', '.'],
+        ['.', '.', '.', '.', '.', '.']]
+
+for first_loop in range(0, len(grid[0])):
+    for second_loop in range(0, len(grid)):
+        print(grid[second_loop][first_loop], end="")
```

Comment:
I would avoid using the additional keyword argument 'end' in print method if not necessary. It's also not supported by python < 3

```
for first_loop in range(0, len(grid[0])):
    line = ''
    for second_loop in range(0, len(grid)):
        line += grid[second_loop][first_loop]
    print(line)
```